@xen-orchestra/lite/src/stories/web-core/tree/tree-loading-item.story.vue

```diff
@@ -0,0 +1,15 @@
+<template>
+  <ComponentStory v-slot="{ properties }" :params="[iconProp().preset(faDisplay).required().widget()]">
```

Comment:
`iconProp()` already configures a widget (a selector with a list of icons).

By calling `.widget()` here, it overrides this configuration and displays nothing.

```suggestion
  <ComponentStory v-slot="{ properties }" :params="[iconProp().preset(faServer).required()]">
```